src/ngraph/serializer.cpp

```diff
@@ -2326,6 +2326,9 @@ shared_ptr<Node> JSONDeserializer::deserialize_node(json node_js)
         case OP_TYPEID::RegionYolo: { break;
         }
         case OP_TYPEID::ReorgYolo: { break;
```

Comment:
Not sure if I should have added the serialization/deserialization when implementing the visitor API for this op. What is the current approach?